recipes/photospline/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "2.0.3" %}
+
+package:
+  name: photospline
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/cnweaver/photospline/archive/v{{ version }}.tar.gz
+  sha256: 7045a631c41489085037b05fac98fd9cad73dc4262b7eead143d09e5f8265dec
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cmake >=3.1
+    - {{ compiler('c') }}  # [not osx]
+    - {{ compiler('cxx') }}  # [not osx]
+  host:
+    - python
+    - numpy >=1.7
+    - suitesparse
+    - cfitsio
+  run:
+    - {{ pin_compatible('python') }}
+    - {{ pin_compatible('numpy', lower_bound='1.7') }}
+    # see: https://github.com/conda-forge/suitesparse-feedstock/issues/22
+    - {{ pin_compatible('suitesparse', max_pin='x.x.x') }}
+    - cfitsio
```

Should this use [`run_exports`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=run_exports#export-runtime-requirements)?
```suggestion
    - cfitsio
  run_exports:
    - {{ pin_subpackage('photospline', max_pin='x.x') }}
```

(I don't know if `x`, `x.x` or `x.x.x` is appropriate for the pinning)

---

Sorry this should have been under `build` rather than `requirements`.